test_app.py

```diff
@@ -40,7 +40,7 @@ def test_consistent_data():
             for lab in labs.values()
             for p in lab['projects'].values()
             ]:
-        # All projects in the incubator have a C4DT contact and a description of work
+        # All projects in the incubator should have a C4DT contact (or a default exists)
```

Comment:
I removed the part that says "a description of work" since now that field is a mandatory one within the "incubator" schema, so there's no need for a check. In fact the test didn't even exist. (probably inconsistent update - fixed)

Reply:
If I understand correctly, the part `that field is a mandatory one within the "incubator" schema` is wrong, correct?

Reply:
No, it's correct. the field `work` is mandatory within the `incubator` schema.   
In other words, if an incubator field exists, it's mandatory that it contains a `work` field.

Reply:
Thanks - I was still fixated on the `contact` field. So all is good.